src/panels/lovelace/hui-view.js

```diff
@@ -17,6 +19,12 @@ class HUIView extends PolymerElement {
         position: relative;
       }
 
+      #badges {
+        margin: 8px 16px;
```

Comment:
This is not good. This means that we're taking this space even if no badges are given. We should hide the div if no badges.